src/test/java/tests/CartTests.java

```diff
@@ -0,0 +1,22 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CartTests extends BaseTests {
+    @Test
+    public void removeItemCartTest() {
+        productsPage.openPage().waitForPageOpened();
+        productsPage.clickAddOrRemoveProductItem(PRODUCT_NAME, true);
+        cartPage.openPage().waitForPageOpened();
+        cartPage.clickRemoveBtn();
```

Comment:
make test in 'Chain of invocations' style:

```
        productsPage.openPage()
                .waitForPageOpened()
                .clickAddOrRemoveProductItem(PRODUCT_NAME, true);
        cartPage.openPage().
                waitForPageOpened().
                clickRemoveBtn();
```

Reply:
done